.github/workflows/pr-auto-label.yml

```diff
@@ -28,7 +28,7 @@ jobs:
             release-note-none
       - name: Add label to PR
         id: labeler
-        uses: jimschubert/labeler-action@v1
+        uses: jimschubert/labeler-action@v2
```

Comment:
@SachinSahu431, could you take a look if this is safe to do? 🙂

Reply:
We might want to append the commit SHA here as well to [avoid backdoors](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#:~:text=Pin%20actions%20to,a%20repository%20fork.)